### PR TITLE
Update error messages

### PR DESCRIPTION
**Pull request by duckofyork**
# Why is this PR needed?
Resource is called attribute now.
# What does the PR do?
Update error messages
# Does it break backwards compatibility?
No
# List of future improvements not on this PR
NA

### CODE REVIEW

File: grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java [hunk 1]
```diff
@@ -52,15 +52,15 @@
...
+    HAS_INVALID("The type [%s] is not allowed to have an attribute of type [%s]"),
     BACKEND_EXCEPTION("Backend Exception."),
     INITIALIZATION_EXCEPTION("Graph for keyspace [%s] not properly initialized. Missing keyspace name resource"),
     TX_CLOSED("The Transaction for keyspace [%s] is closed"),
     SESSION_CLOSED("The session for graph [%s] was closed"),
     TX_CLOSED_ON_ACTION("The transaction was %s and closed [%s]. Use the session to get a new transaction for the graph."),
     TXS_OPEN("Closed session on graph [%s] with [%s] open transactions"),
     LOCKING_EXCEPTION("Internal locking exception. Please clear the transaction and try again."),
-    CANNOT_BE_KEY_AND_RESOURCE("The Type [%s] cannot have the Attribute Type [%s] as a key and as a resource"),
+    CANNOT_BE_KEY_AND_RESOURCE("The Type [%s] cannot have the Attribute Type [%s] as a key and as an attribute"),
```
Comment (lolski) [inline grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java]: LEt's call the enum `CANNOT_BE_KEY_AND_ATTRIBUTE` instead

File: grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java [hunk 2]
```diff
@@ -123,9 +123,9 @@
 
     VALIDATION_RULE_INVALID_RELATION_TYPE("Attempting to define a rule containing a relation pattern with type [%s] which is not a relation type\n"),
 
-    VALIDATION_RULE_INVALID_RESOURCE_TYPE("Attempting to define a rule containing a resource pattern with type [%s] which is not a resource type\n"),
+    VALIDATION_RULE_INVALID_RESOURCE_TYPE("Attempting to define a rule containing an attribute pattern with type [%s] which is not an attribute type\n"),
```
Comment (lolski) [inline grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java]: `VALIDATION_RULE_INVALID_ATTRIBUTE_TYPE`

File: grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java [hunk 3]
```diff
@@ -123,9 +123,9 @@
...
 
-    VALIDATION_RULE_RESOURCE_OWNER_CANNOT_HAVE_RESOURCE("Attempting to define a rule containing a resource pattern of type [%s] with type [%s] that cannot have this resource\n"),
+    VALIDATION_RULE_RESOURCE_OWNER_CANNOT_HAVE_RESOURCE("Attempting to define a rule containing an attribute pattern of type [%s] with type [%s] that cannot have this attribute\n"),
```
Comment (lolski) [inline grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java]: `VALIDATION_RULE_ATTRIBUTE_OWNER_CANNOT_HAVE_ATTRIBUTE`

**Comment by duckofyork** [comment]
Fixed!

**Pull request merged**